cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilder.java

```diff
@@ -968,6 +995,38 @@ private void configureQuotasPluginStrimzi(String clusterName, QuotasPluginStrimz
         writer.println(String.format("client.quota.callback.static.excluded.principal.name.list=%s", String.join(";", excludedPrincipals)));
     }
 
+    /**
+     * Create or update configuration value.
+     *
+     * @param config Configuration.
+     * @param key Property key.
+     * @param value Property value to add or replace.
+     */
+    static void createOrUpdateConfigValue(AbstractConfiguration config, String key, String value) {
+        if (config != null && key != null && !key.isBlank() && value != null && !value.isBlank()) {
+            String existingConfig = config.getConfigOption(key);
+            if (existingConfig == null || !existingConfig.equals(value)) {
+                config.setConfigOption(key, value);
+            }
+        }
+    }
+
+    /**
+     * This method creates a configuration if it does not exist or adds a value to the list if it does exist.
+     *
+     * @param config Configuration.
+     * @param key Property key.
+     * @param value Property value to set or add.
+     */
+    static void createOrAddConfigListValue(AbstractConfiguration config, String key, String value) {
```

Comment:
I would highlight in the parameter name that `config` is a list where you are going to append the value, or?

Reply:
I agree with Paolo, you should rename `config` to `configList`.

Reply:
It is not a list. It is essentially a wrapped properties class!

Reply:
Right, my bad. No need to change here.

Reply:
Ok!!!